src/utilities/router/Router.tsx

```diff
@@ -4,6 +4,7 @@ import { EventGroup } from '../eventGroup/EventGroup';
 export interface IRouterProps {
   replaceState?: boolean;
   children?: React.ReactElement<any>[];
+  routerDidMount?: Function;
```

Comment:
onRouteChange?: () => void;

Avoid a prop of type "Function", a specific empty function that returns nothing is a little more explicit.